packages/grafana-runtime/src/analytics/plugins/eventProperties.ts

```diff
@@ -0,0 +1,35 @@
+import { DataSourceApi, PluginMeta, PluginType } from '@grafana/data';
+
+import { config } from '../../config';
+
+export type PluginEventProperties = {
+  grafana_version: string;
+  plugin_type: PluginType;
+  plugin_version: string;
+  plugin_id: string;
+  plugin_name: string;
+};
+
+export function createPluginEventProperties(meta: PluginMeta): PluginEventProperties {
+  return {
+    grafana_version: config.buildInfo.version,
+    plugin_type: meta.type,
+    plugin_version: meta.info.version,
+    plugin_id: meta.id,
+    plugin_name: meta.name,
```

Comment:
why snake case?

Reply:
That's per [Rudderstack convention](https://docs.google.com/document/d/1MTS5O5qsxtDptzN6AvdMRRGP38pvFc1NULsQKQN_z2U/edit?usp=sharing) I believe

Reply:
Yes, going through the code it looks like we are mixing camel case and snake case for the events. Do we have any "standard" that we can follow here @torkelo?

Reply:
Yes, agree - but this is the data sent via echo service to the configured receivers. So I guess follow what ever the convention is mostly used in analytics makes sense. But I am happy to change to camel case if we don't have any arguments against that.